tests/test_transforms/test_integration.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+import torch.nn as nn
+from compressed_tensors.transforms import Hadamard, RandomHadamard, Transforms
+from compressed_tensors.transforms.transform_args import (
+    ModuleTarget,
+    TransformationArgs,
+)
+from compressed_tensors.transforms.transform_config import TransformationConfig
+from compressed_tensors.transforms.transform_data import TransformData
+from compressed_tensors.transforms.transform_scheme import TransformationScheme
+
+
+@pytest.fixture
+def transform_recipe_basic():
+    targets = ["Linear"]
+    module_targets = [ModuleTarget.WEIGHTS]
+    linear_layer_args = TransformationArgs(
+        targets=targets, module_targets=module_targets
+    )
+
+    scheme = TransformationScheme(
+        transform_type="hadamard",
+        groups=[linear_layer_args],
+        transform_creation_args={"size": 64},
+    )
+    config = TransformationConfig(
+        transform_groups={
+            "transform_0": scheme,
```

Comment:
can this just be `dict[int,TransformationScheme]`? I found this confusing when setting up a `config_group` in the `QuantizationConfig`. Another option is aliasing a TransformId of type int
```python
TransformId = int
class TransformationConfig(BaseModel):
    transform_groups: dict[TransformId, TransformationScheme]
```